pkg/discovery/worker/compliance/unschedulable_node_anti_affinity_group_dto_builder.go

```diff
@@ -0,0 +1,102 @@
+package compliance
+
+import (
+	"fmt"
+	"github.com/golang/glog"
+	"github.com/turbonomic/kubeturbo/pkg/discovery/repository"
+	"github.com/turbonomic/turbo-go-sdk/pkg/builder/group"
+	"github.com/turbonomic/turbo-go-sdk/pkg/proto"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+// In Turbo, Schedulable nodes are marked with an access commodity 'schedulable'. Pods are required to buy
+// this commodity so they can be moved to schedulable nodes
+// Unchedulable nodes do not sell this commodity, hence pods will not be moved to these nodes.
+// We want to address the issue that pods that are already running on the unschedulable nodes are not moved out
+// due to compliance issues. This is done by not requiring the pods already on the unschedulable nodes at the tine
+// of discovery to not buy the 'schedulable' commodity so they can continue staying on the current node unless there
+// are resource constraints or unless moved by the kubernetes controller.
+// In addition, we need pods which do not buy the schedulable commodity to not move to other unschedulable nodes.
+// This is achieved by creating an anti-affinity policy for these pods to the other unschedulable nodes other than the
```

Comment:
Nit: Probably replace to: `This is achieved by creating an anti-affinity policy for pods running on this unschedulable node against all other unschedulable nodes.`